src/server.c

```diff
@@ -5776,6 +5789,8 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "maxmemory:%lld\r\n", server.maxmemory,
                 "maxmemory_human:%s\r\n", maxmemory_hmem,
                 "maxmemory_policy:%s\r\n", evict_policy,
+                "maxmemory_reserved:%lld\r\n", server.maxmemory_reserved,
+                "maxmemory_reserved_human:%s\r\n", maxmemory_reserved_hmem,
```

Comment:
```suggestion
                "maxmemory:%lld\r\n", server.maxmemory - server.maxmemory_reserved,
                "maxmemory_human:%s\r\n", maxmemory_reserved_hmem,
                "maxmemory_policy:%s\r\n", evict_policy,
```
I would like to consider this. Since this is effectively the memory we are making available to the end users for their data, and where we start evicting based off of. Adding another config will likely cause more confusion.

Reply:
I do not like to do the mathematics as well. Agree with you.  I will update it after we finalize the naming for this pr. Thanks

Reply:
Users still need to do some math to know when eviction happens. `mem_not_counted_for_evict` is part of the formula.

The math will be easier for users if we add a field like `free_memory_below_eviction_threshold` defined (if my math is right) like this:

    maxmemory - maxmemory_reserved - used_memory + mem_not_counted_for_evict

With a field like this, users can just check this to know how near eviction is.

I remember this doc PR and linked issue: https://github.com/valkey-io/valkey-doc/pull/194. It's the problem for users to understand this math.